pyproject.toml

```diff
@@ -41,6 +41,7 @@ pytest = "^8.2.0"
 chromadb = "^0.5.0"
 sumy = "^0.11.0"
 fake-useragent = "^1.5.1"
+youtube-transcript-api = "^0.6.2"
```

Comment:
Please remove this package. These additional packages must be installed ondemand (just before using the respective tools) based on the usecases being ran.

Reply:
@shreehari-aiplanet added youtube loader, like user can pass video url and based on that worker fetch the information and utilise it. I will remove this and comment the youtube loader part.